bin/hdfcoinc/pycbc_page_snrifar

```diff
@@ -198,12 +283,25 @@ if not args.closed_box:
                 # make the arrow length 1/15 the height of the plot
                 arr_end = arr_start * (plot_ymin / plot_ymax) ** (1./15)
                 pylab.plot([r, r], [arr_start, arr_end], lw=2, color='black',
-                    zorder=99)
+                           zorder=99)
                 pylab.plot([r, r], [arr_start, arr_end], lw=2.6, color='white',
-                    zorder=97)
+                           zorder=97)
                 pylab.scatter([r], [arr_end], marker='v', c='black',
-                    edgecolors='white', lw=0.5, s=40, zorder=98)
-            
+                              edgecolors='white', lw=0.5, s=40, zorder=98) 
+
+            if h_inc_back_num > 0:
+               for ii in range(0, len(cstat_fore_h_rm)):
+                   r = cstat_fore_h_rm[ii]
+                   arr_start = cstat_rate_h_rm[ii]
+                   # make the arrow length 1/15 the height of the plot
+                   arr_end = arr_start * (plot_ymin / plot_ymax) ** (1./15)
+                   pylab.plot([r, r], [arr_start, arr_end], lw=2,
+                              color='black', zorder=99)
+                   pylab.plot([r, r], [arr_start, arr_end], lw=2.6,
+                              color='white', zorder=97)
+                   pylab.scatter([r], [arr_end], marker='v', c='black',
+                                 edgecolors='white', lw=0.5, s=40, zorder=98)
```

Comment:
This is a fair critique, I think Alex Nitz made a patch that made arrows point to events that were louder in ranking statistics rather than how it was before (via FAR).